### ISSUE TITLE
Feature: Include server name in profile download

### ISSUE THREAD

**Issue by mlanner** [ngoduykhanh/wireguard-ui#386]
For this also see:

https://github.com/ngoduykhanh/wireguard-ui/issues/385

When I download a profile from the WireGuard Clients page, it would be great if it had some kind of indicator of what instance it is for. For example, if I have a user **John Smith** with a profile name **jsmith**, if I have that same user on **wg-server01** and **wg-server02**, without opening the conf files and cross-referencing the IP/FQDN set, I can't tell which profile goes with what server.

So, in short, it would be fantastic if when I downloaded a profile, it would download as **wg-server01_jsmith.conf**, and so on.
